docs/transfer-verifier.md

```diff
@@ -0,0 +1,16 @@
+# Transfer Verifier
+
+Below are admin controls for the Guardians to configure the Transfer Verifier plugin. For
+a background on the feature and its objectives, see [the whitepaper](whitepapers/0014_transfer_verifier.md)
+
+## How To Enable Transfer Verifier
+The Transfer Verifier feature is disabled by default. Guardians can enable it by passing the following flag to the `guardiand` command when starting it up:
+
+```bash
+# Enable Transfer Verifier for chain with ID 2 (Ethereum)
+--transferVerifierEnabledChainIDs=2
+```
+
+This parameter is a comma-separated list of Wormhole Chain IDs for which transfer verification will be enabled.
+
+Only some chains support the Transfer Verifier.
```

Comment:
```suggestion
Only some chains support the Transfer Verifier. Enabling Transfer Verifier for Chain IDs via the command line that do not have the underlying support will have no effect. 
```

Reply:
This suggestion isn't accurate. It will cause the node to fail to start.

https://github.com/wormhole-foundation/wormhole/blob/bce6b859f4f95b012619d08fcf84394885fa11d8/node/cmd/guardiand/node.go#L960-L968

Reply:
Oh I misremembered! Could you add the inverse of what I suggested then? (i.e. what you described in your comment)

Reply:
```suggestion
Only some chains support the Transfer Verifier. If an unsupported chain is specified, the node will not start. It will also display an error message stating that the chain does not have a Transfer Verifier implementation.
```